modules/core/src/test/scala/org/scalasteward/core/nurture/UpdateInfoUrlFinderTest.scala

```diff
@@ -178,23 +171,20 @@ class UpdateInfoUrlFinderTest extends CatsEffectSuite with Http4sDsl[MockEff] {
     )
 
     assertEquals(
-      possibleVersionDiffs(AzureRepos, onPremForgeUrl, onPremForgeUrl.addPath("foo/bar"), v1, v2)
+      possibleVersionDiffs(ForgeRepo(AzureRepos, onPremForgeUrl.addPath("foo/bar")), versionUpdate)
         .map(_.url.renderString),
       List(
-        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=v$v1&targetVersion=v$v2",
-        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=$v1&targetVersion=$v2",
-        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=release-$v1&targetVersion=release-$v2"
+        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=GTv$v1&targetVersion=GTv$v2",
+        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=GT$v1&targetVersion=GT$v2",
+        s"${onPremForgeUrl}foo/bar/branchCompare?baseVersion=GTrelease-$v1&targetVersion=GTrelease-$v2"
```

Comment:
Note that the format of Azure 'diff' urls is slightly different to what Scala Steward has been generating up to now - the tag names used in actual Azure urls are prefixed with `GT` on the query parameter values:

https://dev.azure.com/rtyley/scala-steward-testing/_git/scala-steward-testing/branchCompare?baseVersion=GTv1.0.0&targetVersion=GTv1.0.1

Removing the `GT` prefixes will get a `Unrecognized VersionSpec format` error from Azure:

https://dev.azure.com/rtyley/scala-steward-testing/_git/scala-steward-testing/branchCompare?baseVersion=v1.0.0&targetVersion=v1.0.1